tracking/include/pcl/tracking/particle_filter.h

```diff
@@ -592,14 +579,14 @@ class ParticleFilterTracker : public Tracker<PointInT, StateT> {
   std::vector<double> initial_noise_mean_;
 
   /** \brief The threshold for the particles to be re-initialized. */
-  double resample_likelihood_thr_;
+  double resample_likelihood_thr_{0.0};
 
   /** \brief The threshold for the points to be considered as occluded. */
-  double occlusion_angle_thr_;
+  double occlusion_angle_thr_{0.0};
```

Comment:
```suggestion
  double occlusion_angle_thr_{M_PI / 2.0};
```
And remove it from constructor - seems more variables could be member initialized here as well?